plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SqlExtension.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.gradle.spotless;
+
+import static com.diffplug.gradle.spotless.PluginGradlePreconditions.requireElementsNonNull;
+
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.sql.SQLFormatterStep;
+import org.gradle.api.Project;
+
+public class SqlExtension extends FormatExtension {
+	static final String NAME = "sql";
+
+	public SqlExtension(SpotlessExtension rootExtension) {
+		super(rootExtension);
+	}
+
+	public SQLFormatterConfig sqlFormatter() {
+		return new SQLFormatterConfig();
```

Comment:
The name here should probably match the step name.  So if we name the step `dbeaverSql`, then we should probably name this method `dbeaver`.  We might add other sql formatters later, so its best to name them fairly specifically. (we've got a couple different java formatters right now, and we might have a few different Kotlin formatters soon too).

Reply:
Oh, also run `spotlessApply` ;-)